src/Protocol/DFRN.php

```diff
@@ -1585,19 +1644,34 @@ private static function fetchauthor($xpath, $context, $importer, $element, $only
 			if ($update) {
 				logger("Update contact data for contact " . $contact["id"] . " (" . $contact["nick"] . ")", LOGGER_DEBUG);
 
-				q("UPDATE `contact` SET `name` = '%s', `nick` = '%s', `about` = '%s', `location` = '%s',
+				q(
+					"UPDATE `contact` SET `name` = '%s', `nick` = '%s', `about` = '%s', `location` = '%s',
 					`addr` = '%s', `keywords` = '%s', `bdyear` = '%s', `bd` = '%s', `hidden` = %d,
 					`xmpp` = '%s', `name-date`  = '%s', `uri-date` = '%s'
 					WHERE `id` = %d AND `network` = '%s'",
-					dbesc($contact["name"]), dbesc($contact["nick"]), dbesc($contact["about"]), dbesc($contact["location"]),
-					dbesc($contact["addr"]), dbesc($contact["keywords"]), dbesc($contact["bdyear"]),
-					dbesc($contact["bd"]), intval($contact["hidden"]), dbesc($contact["xmpp"]),
-					dbesc(dbm::date($contact["name-date"])), dbesc(dbm::date($contact["uri-date"])),
-					intval($contact["id"]), dbesc($contact["network"]));
+					dbesc($contact["name"]),
+					dbesc($contact["nick"]),
+					dbesc($contact["about"]),
+					dbesc($contact["location"]),
+					dbesc($contact["addr"]),
+					dbesc($contact["keywords"]),
+					dbesc($contact["bdyear"]),
+					dbesc($contact["bd"]),
+					intval($contact["hidden"]),
+					dbesc($contact["xmpp"]),
+					dbesc(DBM::date($contact["name-date"])),
+					dbesc(DBM::date($contact["uri-date"])),
+					intval($contact["id"]),
+					dbesc($contact["network"])
```

Comment:
As crazy as it is, this formatting made sense because the parameters lines followed the query lines.

A better way to improve readability would be to use `dba::update()`

Reply:
This was again just done based on suggestion from the tool. Something about calls being on their own line.